common/tensorrt_common/src/tensorrt_common.cpp

```diff
@@ -427,18 +427,18 @@ bool TrtCommon::buildEngineFromOnnx(
             layer->setPrecision(nvinfer1::DataType::kHALF);
             std::cout << "Set kHALF in " << name << std::endl;
           }
-          for (int i = num - 1; i >= 0; i--) {
-            nvinfer1::ILayer * layer = network->getLayer(i);
-            auto layer_type = layer->getType();
-            std::string name = layer->getName();
-            if (layer_type == nvinfer1::LayerType::kCONVOLUTION) {
+          for (int j = num - 1; j >= 0; j--) {
+            nvinfer1::ILayer * inner_layer = network->getLayer(j);
+            auto inner_layer_type = inner_layer->getType();
+            std::string inner_name = inner_layer->getName();
+            if (inner_layer_type == nvinfer1::LayerType::kCONVOLUTION) {
               layer->setPrecision(nvinfer1::DataType::kHALF);
```

Comment:
Do you mean `inner_layer` ?

Reply:
That is correct. It was a careless mistake. I will be careful from now on.

Reply:
Corrected.